app/assets/stylesheets/registration3.scss

```diff
@@ -463,12 +463,60 @@ body {
   }
 }
 
+/* CTW additions */
+
+#main_inner .declaration-text {
+  background-color: #F5EEDD;
+  padding:10px;
+  margin:20px;
+  line-height:1.5;
+}
+
+
+.newui2020 #main_inner form ul.checkbox li h3 {
```

Comment:
registration3 should only load for users using the new UI so we don't need to explicitly target that.

Reply:
Truth

Reply:
Done